packages/experimental/TabList/src/TabList/TabList.tsx

```diff
@@ -65,7 +69,11 @@ export const TabList = compose<TabListType>({
           value={tabs.state.context}
         >
           <Slots.root {...mergedProps}>
-            <Slots.container defaultTabbableElement={defaultTabbableElement} isCircularNavigation={isCircularNavigation}>
+            <Slots.container
+              disabled={final.disabled}
+              defaultTabbableElement={defaultTabbableElement}
+              isCircularNavigation={isCircularNavigation}
+            >
```

Comment:
I know this PR is for tokens/styling but when I was keyboarding, I noticed that the tablist is focusable. Is this by design? So for example, when you have keyboard focus on a tab and tab to the next element, focus goes to the next tablist (no focus indicator shown) then the tab in that tablist.

Reply:
It's actually not intended. I have local changes that fix this, but I don't want to increase the scope of this PR.